plugins/hydra_optuna_sweeper/setup.py

```diff
@@ -27,7 +27,7 @@
     ],
     install_requires=[
         "hydra-core>=1.1.0.dev7",
-        "optuna>=2.10.0",
```

Comment:
could you keep the "optuna>=2.10.0"  in the meanwhile?

Reply:
So you mean it's enough to track the issue #2162 and not necessary to constrain the version, right? It will show warning messages for users when Optuna v3 is available. Is it OK?

Reply:
Just throwing this out there: using multiple constraints on the version should be legal:
```python
    install_requires=[
        ...
        "optuna >= 2.10.0, < 3.0.0",
        ...
```
The downside of pinning Optuna < 3.0 is that users would not be able to access new Optuna features. Jieru, is why you were concerned about the pin?

Reply:
sorry for the confusion, i should have provvided more context.  I meant `"optuna >= 2.10.0, < 3.0.0"`

the current impl assumes the version is >=2.10.0 so it'd be great to keep that still

Reply:
Thank you @Jasha10 and @jieru-hu! Sorry for my poor understanding. 4b39011 restore the lower constraint. 🙋

Reply:
thank you @himkt!